notebooks/experiments/conditional_diffusion/dna_diff_baseline_conditional_UNET.ipynb

```diff
@@ -0,0 +1,7119 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    def log(t, eps = 1e-20):
</pre><p>We should add the <code>torch.Tensor</code> type hint to the parameter <code>t</code> and the function output.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pinellolab/DNA-Diffusion/pull/60/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>